core/modules/replica/ControllerApp.cc

```diff
@@ -370,6 +577,10 @@ ControllerApp::ControllerApp(int argc, char* argv[])
     parser().command("SERVICE_DRAIN").description(
         "Drain all requests by stopping cancelling all ongoing requests"
         " and emptying all queues.");
+
+    parser().command("SERVICE_RECONFIG").description(
+        "Reload worker's Configuration. Requests known to a worker won't be affected"
+        " by the operation.");
```

Comment:
parser setup is almost 400 lines long. Maybe it could be factored out of the ctor for readability?